### PR TITLE
Changes in backdrop, changes in linking to movie

### PR DESCRIPTION
**Pull request by lkkolosowski**
I wanted to change the inner shadow in backdrop. What do you think about this change? 🤔

### CODE REVIEW

**Comment by lkkolosowski** [comment]
moved linking from entire movie tile to movie image only

File: src/features/MoviesList/styled.js [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-  & > li:hover {
+  &:hover {
     transform: translateY(-12px);
   }
 
   @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    gap: 16px;
-    grid-template-columns: 1fr 1fr;
-    & > li:hover {
+    &:hover {
```
Comment (Albrecht-Albi) [inline src/features/MoviesList/styled.js:29]: Good idea with the media query, you can immediately see the style text at low resolution, we won't come back to that anymore.

File: src/features/MoviesList/index.js [hunk 2]
```diff
@@ -21,23 +20,21 @@ const MoviesList = () => {
         content={
           <>
             <Title title="Popular Movies"></Title>
-            <Wrapper>
+            <List>
```
Comment (Albrecht-Albi) [inline src/features/MoviesList/index.js:23]: This name is a better fit for this file

File: src/features/MovieDetails/index.js [hunk 3]
```diff
@@ -33,29 +29,17 @@ const MovieDetails = () => {
...
-              </Backdrop>
-            }
-          />
-        </BackdropWrapper>
+        <Backdrop
+          background={`${URL}${details.backdrop_path}`}
+          title={details.original_title}
+          vote={details.vote_average}
+          votes={details.vote_count}
+        />
```
Comment (Albrecht-Albi) [inline src/features/MovieDetails/index.js:37]: This code looks much better than the last, cool idea for modifications

File: src/features/MovieDetails/Backdrop/index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                votes={votes}
+              />
+            )}
+          </BackdropContent>
+        </Overlay>
+      </BackdropBody>
+    </Wrapper>
+  </BackdropWrapper>
+);
+
```
Comment (Albrecht-Albi) [inline src/features/MovieDetails/Backdrop/index.js:30]: You have created a new component well. now the job looks more complete

**Review by Albrecht-Albi — APPROVED** [review]

**Pull request merged**